database_updates.php

```diff
@@ -1374,12 +1374,26 @@
         mysqli_query($mysqli, "UPDATE `settings` SET `config_current_database_version` = '0.8.6'");
     }
 
-    //if (CURRENT_DATABASE_VERSION == '0.8.6') {
+    // Update DB to 0.8.7
+    if (CURRENT_DATABASE_VERSION == '0.8.6') {
     // Insert queries here required to update to DB version 0.8.7
+    mysqli_query($mysqli, "ALTER TABLE `accounts` ADD `account_type` int(6) DEFAULT NULL AFTER `account_notes`");
```

Comment:
Should this default to '11', like db.sql?

Reply:
No, the default should be null. Good catch.